src/main/java/com/stripe/model/Account.java

```diff
@@ -1199,6 +1202,37 @@ public void setFrontObject(File expandableObject) {
       @SerializedName("creditor_id")
       String creditorId;
     }
+
+    @Getter
+    @Setter
+    @EqualsAndHashCode(callSuper = false)
+    public static class SettingsCardIssuing extends StripeObject {
```

Comment:
Every time I see those I get sad, they should have been nested inside `Settings` instead but we missed fixing them originally and then got stuck